src/packagedcode/npm.py

```diff
@@ -829,7 +829,11 @@ def build_packages_from_lockfile(package_data):
     for dependency, values in package_data.get('dependencies', {}).items():
         is_dev = values.get('dev', False)
         dep_deps = []
-
+        namespace = None
+        name = dependency
+        if '/' in dependency:
+            namespace = dependency.partition('/')[0]
+            name = dependency.partition('/')[2]
```

Comment:
You could write instead a single statement to get both ns and name at once such as with this:
`namespace, _slash, name = dependency.partition('/')`